test/integration_test.dart

```diff
@@ -852,9 +885,14 @@ void main() {
         await tester.pumpWidget(buildApp());
         await tester.pumpAndSettle();
 
-        // Open faq page
-        final showMoreFaq = find.byKey(const ValueKey('show_more_faq'));
+        final showMoreFaq =
```

Comment:
Don't delete the `// Open faq page` comment

Reply:
I just moved it a few lines lower, where the actual opening happens.

Reply:
Ahh, fair point